readme.md

```diff
@@ -62,6 +62,8 @@ The `distribution` block offers the following options:
  * (optional) `checkArgs` a list of arguments to supply to the monitoring script, if omitted,
    no monitoring script will be generated.
  * (optional) `defaultJvmOpts` a list of default JVM options to set on the program.
+ * (optional) `customClasspath` a list of jars to put on the classpath instead of all jars in the
```

Comment:
this is not quite complete enough. at the minimum, must say that these are file names relative to some path, that they must not contain certain characters, something about the classpath order